### PR TITLE
Add support for `source` in signals

### PR DESCRIPTION
**Pull request by syyyr**

### CODE REVIEW

File: libshvbroker/src/brokerapp.cpp [hunk 1]
```diff
@@ -857,7 +857,7 @@ void BrokerApp::checkLogin(const chainpack::UserLoginContext &ctx, const QObject
 void BrokerApp::sendNewLogEntryNotify(const std::string &msg)
 {
 	if(isSendLogAsSignalEnabled())
-		sendNotifyToSubscribers(".broker/app/log", cp::Rpc::SIG_VAL_CHANGED, msg);
+		sendNotifyToSubscribers(".broker/app/log", cp::Rpc::SIG_VAL_CHANGED, "get", msg);
```
Comment (fvacek) [inline libshvbroker/src/brokerapp.cpp:860]: `cp::Rpc::METHOD_GET`

File: libshvchainpack/include/shv/chainpack/irpcconnection.h [hunk 2]
```diff
@@ -45,10 +45,10 @@ class SHVCHAINPACK_DECL_EXPORT IRpcConnection
 	int callShvMethod(int rq_id, const std::string &shv_path, std::string method, const shv::chainpack::RpcValue &params = shv::chainpack::RpcValue());
 	int callShvMethod(int rq_id, const std::string &shv_path, std::string method, const shv::chainpack::RpcValue &params, const shv::chainpack::RpcValue &user_id);
 
-	int callMethodSubscribe(const std::string &shv_path, std::string method);
-	int callMethodSubscribe(int rq_id, const std::string &shv_path, std::string method);
-	int callMethodUnsubscribe(const std::string &shv_path, std::string method);
-	int callMethodUnsubscribe(int rq_id, const std::string &shv_path, std::string method);
+	int callMethodSubscribe(const std::string &shv_path, std::string method, const std::string& source = "");
```
Comment (fvacek) [inline libshvchainpack/include/shv/chainpack/irpcconnection.h:48]: default source should be IMO `get`

**Review by fvacek — CHANGES_REQUESTED** [review]

File: libshvchainpack/include/shv/chainpack/irpcconnection.h (hunk 2 above)
Comment (syyyr) [reply libshvchainpack/include/shv/chainpack/irpcconnection.h:48]: I don't think it should, by default, `callMethodSubscribe` should subscribe to all the sources. Or I could make the argument mandatory. But this way, older code will be still compatible, and do the same thing.

Comment (fvacek) [reply libshvchainpack/include/shv/chainpack/irpcconnection.h:48]: agree

**Pull request merged**